functions.php

```diff
@@ -18,15 +18,15 @@
 		'lib/cleaner/wordpress.php',  // Clean WordPress things
 		'lib/cleaner/wpml.php',       // Clean WPML things
 		'lib/cleaner/yoast.php',      // Clean Yoast things
-
+		
 		// Custom Post Types
```

Comment:
Why do we import Custom Post Types and Metaboxes via another file and taxonomies not? Change it, please. Also, remove comments and empty lines and keep only inline comments describing that line.